server/controllers/chartsController.js

```diff
@@ -249,3 +244,13 @@ function isAnyTargetIncluded(studyTotals) {
     .filter((site) => site !== TOTALS_STUDY)
     .some((site) => studyTotals[site]?.targetTotal !== undefined)
 }
+
+function isTargetValueOnAllSites(fieldLabelValueMap, valueLabel) {
+  return Object.values(
+    fieldLabelValueMap.find(({ label }) => label === valueLabel).targetValues
+  ).every((target) => !!target)
```

Comment:
Minor suggestion to make this more readable.

```suggestion
  const fieldTargetValues = fieldLabelValueMap.find(({ label }) => label === valueLabel).targetValues
  
  return Object.values(fieldTargetValues).every(Boolean)
```

Reply:
[cbdb19e](https://github.com/AMP-SCZ/dpdash/pull/217/commits/cbdb19ee4098d9b8d76793cb0d383fe9ad4e86d7)